app/views/interactive_pages/_show.html.haml

```diff
@@ -2,14 +2,18 @@
   - unless page.name.blank?
     %h4.h4
       = page.name
-  - if page.show_introduction && !page.text.blank?
-    .intro-txt!= page.text
+  - if page.show_header
+    .header-block-mod
+      -# Wrapping plugins don't necessarily wrap items in the assessment block, so they need to be rendered separately.
+      = render partial: 'interactive_pages/list_wrapping_plugins', locals: {wrapping_plugins: main_section_wrapping_plugins(page, @run)}
```

Comment:
This seems to be rendering the main section wrapping plugins inside of the header block. I think this will mean the main section plugins will be rendered twice.